build/npm/waypoint-hoc.js

```diff
@@ -218,6 +218,11 @@ var waypoint = function waypoint(Component) {
           });
         }
       }
+
+      /**
+       * @return {Object} `height` and `top` of scrollable ancestor Rect.
+       */
+
     }, {
       key: '_scrollableAncestorHeightTop',
       value: function _scrollableAncestorHeightTop() {
```

Comment:
Files in the `build` folder are in the `.gitignore` file, so they shouldn't be commited.